cybercom_queue/views.py

```diff
@@ -134,8 +134,10 @@ def post(self, request, task_name=None, format=None):
         if type(kwargs) is not dict:
             return Response({'error': 'kwargs must be a JSON object'})
         tags = request.data.get('tags', [])
+        user_info = request.user.__class__.objects.filter(pk=request.user.id).values().first()
+        user_info.pop('password', None)
```

Comment:
Does this have the potential to leak credentials?

Reply:
I do not believe that it does.... I'm removing the password and passing through encrypted communication to rabbitmq--> celery task. I'm open to any thoughts or updated code. The request user does not have the plain text password but wanted to remove any security vulnerabilities.